frontends/common/parser_options.cpp

```diff
@@ -395,10 +395,13 @@ bool ParserOptions::searchForIncludePath(const char *&includePathOut,
 }
 
 std::vector<const char *> *ParserOptions::process(int argc, char *const argv[]) {
-    searchForIncludePath(p4includePath, {"p4include"_cs, "../p4include"_cs, "../../p4include"_cs},
+    searchForIncludePath(p4includePath,
+                         {"p4include"_cs, "../p4include"_cs, "../../p4include"_cs,
```

Comment:
Maybe create a virtual function that returns this list? Then the list can be overridden by other back ends.